### DISCUSSION TITLE
Scrutini altri periodi

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#480] [Richieste di aiuto] [answered]
Buonasera,
utilizzando il registro nella mia Scuola è sorta una problematica: questa, infatti, prevede la suddivisione dell'anno in trimestre e pentamestre, ma il pentamestre è a sua volta diviso da una valutazione interperiodale (pagellino), a fine marzo, che non ha valenza giuridica ma solo informativa alle famiglie.
Il problema a questo punto è come poter effettuare questo scrutinio interperiodale dal registro e come trasmettere il documento alle famiglie.
Spero che possa dami qualche suggerimento in merito, o qualche altra strada da percorrere.
Io avevo pensato all'utilizzo del terzo periodo indicato dal registro, ma in quel modo le valutazioni del periodo finale sarebbero separate da quelle del secondo periodo, il che non va bene.
In alternativa mi chiedevo se fosse possibile, in qualche modo, caricare nella sezione scrutini un documento con le valutazioni per le varie discipline, o aggiungere lo scrutinio di un periodo, ma mi rendo conto che non è una procedura semplice.
Mi faccia sapere.
Saluti.

**Comment by trinko** [comment, marked as answer]
Il problema non è semplice da risolvere perché questa gestione non è proprio prevista.
Come dicevi, creare un terzo periodo può creare problemi perché avresti tutto il registro suddiso formalmente nei tre periodi.

Mi pare che la soluzione più semplice sia creare il documento di valutazione esternamente al registro elettronico. Poi questi documenti si possono inserire come avvisi. Non si possono infatti inserire come documenti di scrutinio perché questi vengono generati a partire dai dati inseriti nello scrutinio.

Il problema che troverai è che in questo modo bisogna creare un avviso con documento allegato per ogni singolo alunno...
Francamente, a questo punto può essere più conveniente stampare la comunicazione e consegnarla a mano.

**Reply by Lucaccc** [reply]
Va bene, grazie mille.
Cercheremo soluzioni con la presidenza.
